### ISSUE TITLE
Balsamic: List of all uploaded variants in a gene when looking at one variant

### ISSUE THREAD

**Issue by KickiLagerstedt** [Clinical-Genomics/scout#2459]
Request to have the same feature as in MIP/SCOUT where all possibly compound variants are uploaded when looking at one variant. Including VAF-data....

![image](https://user-images.githubusercontent.com/17139607/111796517-8b7d1800-88c8-11eb-90fb-b6b56ae2b7d7.png)

**Comment by dnil** [comment]
Absolutely; I think we actually do show them in Scout already, if they are available. I will forward this to Balsamic. Compounds are calculated by Genmod from MIP, and Genmod has recently been added to Balsamic as well, so should be somewhat straightforward. 

When we are at it, is there any change you would like in the mode of operation later? First line, we will assume you are happy with linking any second event as a compound, regardless if inherited or somatic, but it might be possible to e.g. consider VAF thresholds or tumor-normal pairing limiting the actual compound call. In that case we will need to update Genmod (@moonso) a bit as well in a second step.
